src/store/actions.ts

```diff
@@ -436,6 +444,69 @@ export const submit: any = async ({ commit, getters, dispatch }): Promise<any> =
   }
 }
 
+/**
+ * Re-submits an expired NR (without changing the current NR data - in case of failure).
```

Comment:
It was really frustrating to have multiple store objects for a new NR or the current NR (for display)... To avoid messing up that code, this method works with local data and overrides things as needed so the POST data object looks like a new NR request.